README.md

```diff
@@ -42,7 +42,29 @@ class SwitchStencil : ViewStencil(
 }
 ```
 
-The set of `ViewStencil`s that Artist should process are provided via the `ViewStencilProvider`. The [sample's ViewStencilProvider](https://github.com/uber/artist/blob/master/sample/providers/src/main/java/com/uber/artist/myproviders/SampleViewStencilProvider.java) would configure Artist to generate [these Views](https://github.com/uber/artist/tree/master/sample/library/build/generated/source/artist/release/com/uber/artist/mylibrary).
+Finally leaving you with a generated view like this:
+
+```java
+public class MySwitch extends SwitchCompat {
+  // Constructors
+  
+  // protected init method - provided in every stencil
+  
+  public boolean isVisible() {
+    return getVisibility() == View.VISIBLE;
+  }
+  
+  public boolean isGone() {
+    return getVisibility() == View.GONE;
+  }
+  
+  public boolean isInvisible() {
+    return getVisibility() == View.INVISIBLE;
+  }
+}
+```
+
+This may look like a lot of boilerplate for simple helpers, but it scales quite well when you want to have these methods on _all_ your base views.
```

Comment:
maybe we should linkify "all your base views" to the generated views in the sample app? to complete the trait -> stencil -> single view -> all views that Artist is intended for?

Reply:
I don't think so. You're trying to give a concise gist, and last thing you want to do is take them out of this context and make them look somewhere else for that detail.